internal/tokens/fftokens/fftokens.go

```diff
@@ -151,6 +153,17 @@ type activatePool struct {
 	RequestID   string             `json:"requestId,omitempty"`
 }
 
+type tokenInterface struct {
+	Format  core.TokenInterfaceFormat `json:"format"`
+	ABI     []*abi.Entry              `json:"abi,omitempty"`
+	Methods []*fftypes.FFIMethod      `json:"methods,omitempty"`
```

Comment:
I think `ffi` would be less confusing for the name of this parameter if possible.

As on the other end of the token connector where it talks to EVMConnect/EthConnect etc. the `methods` array is going to represent an array of ABI-style methods. Also where `abi` is used above, I think it's _actually_ an array of ABI methods (not the whole ABI).